capstone/src/test/javascript/search/autocompleteSpec.js

```diff
@@ -1,15 +1,16 @@
-describe('Autocomplete Suggests correct array', () => {
-  it('Autocomplete selects correct item', () => {
-    const fullData = ['Ahoy', 'Hello', 'Wattup'];
-    const relevantData = ['Hello'];
-    const retData = getRelevantDataSuggestions(fullData, 'H');
-    expect(retData).toEqual(relevantData);
+describe('Autocomplete provides correct list element', () => {
+  it('Tests Proper List Element Creation Simple', () => {
+    const arrElt = 'Hello';
```

Comment:
can you give these variables more descriptive names? `arrElt` and `val` don't give the reader much to go on...

Reply:
updated!